improver_tests/calibration/ensemble_calibration/test_CalibratedForecastDistributionParameters.py

```diff
@@ -104,42 +116,61 @@ def setUp(self):
         )
         self.coeffs_from_no_statsmodels_realizations = estimator.create_coefficients_cubelist(
             self.expected_realizations_norm_no_statsmodels,
-            self.current_temperature_forecast_cube,
+            self.historic_temperature_forecast_cube,
+        )
+
+        # Set up a coefficients cube when using the ensemble realization as the
+        # predictor and the coefficients have been estimated using statsmodels.
+        expected_realizations_each_site = np.vstack(
+            list(self.expected_realizations_each_site.values())
+        )
+        estimator = EstimateCoefficientsForEnsembleCalibration(
+            "norm", predictor="realizations", each_point=True
+        )
```

Comment:
Depending upon how you decide to proceed with the equivalency of `each_point` and `minimise_each_point` when using realizations as the predictor, you may want to:
- add a test that demonstrates they are equivalent
- if you change something so this is not the case, demonstrate that they give different results.

Reply:
There are now tests for `each_point` and `minimise_each_point` using realizations and these have different results.

Reply:
You're correct that the tests covering both `each_point` and `minimise_each_point` are for the `EstimateCoefficientsForEnsembleCalibration` plugin. I haven't added a test using `minimise_each_point` for the `CalibratedForecastDistributionParameters` plugin, because I think that the difference between the `each_point` and `minimise_each_point` option is really only relevant within the `EstimateCoefficientsForEnsembleCalibration` plugin rather than the  `CalibratedForecastDistributionParameters` plugin.